source/dofs/dof_renumbering.cc

```diff
@@ -967,22 +967,18 @@ namespace DoFRenumbering
     for (unsigned int component = 0; component < fe_collection.n_components();
          ++component)
       {
-        const typename std::vector<types::global_dof_index>::const_iterator
-          begin_of_component = component_to_dof_map[component].begin(),
-          end_of_component   = component_to_dof_map[component].end();
-
         next_free_index = shifts[component];
 
-        for (typename std::vector<types::global_dof_index>::const_iterator
-               dof_index = begin_of_component;
-             dof_index != end_of_component;
-             ++dof_index)
+        for (const types::global_dof_index dof_index :
```

Comment:
clang-tidy might complain if we do not use a reference here.

Reply:
It shouldn't -- it's just an integer type...